style.css

```diff
@@ -129,6 +129,151 @@ label {
     font-size: 1.7rem;
     font-weight: bolder;
   }
+
+  .popups {
+    width: 100vw;
+    background: #c1c7d0;
+    backdrop-filter: blur(10px);
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    z-index: 1001;
+    display: none;
+  }
+
+  .modal-container {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 10;
+    background-color: #fff;
+    width: 80vw;
+    max-width: 90vw;
+    height: 90vh;
+    overflow-y: scroll;
+    overflow-x: hidden;
```

Comment:
Nice job so far👍🏼. The functionalities of your popup works just fine😉. But let's try to make it match the given design both on mobile and on desktop😉.

Design
<img width="660" alt="Screenshot 2023-08-02 at 08 18 25" src="https://github.com/melkamu372/Portfolio-mobile-first/assets/64643995/458f8e74-c59f-428a-8cad-0c4df7d2a6c7">

Yours
<img width="1208" alt="Screenshot 2023-08-02 at 08 28 15" src="https://github.com/melkamu372/Portfolio-mobile-first/assets/64643995/db15cf8e-747e-4b28-a937-72641d5ba17e">
<img width="1208" alt="Screenshot 2023-08-02 at 08 28 33" src="https://github.com/melkamu372/Portfolio-mobile-first/assets/64643995/5622d368-38d2-4f4f-a679-7c6397641f0c">

Take note of the positioning of the elements, their sizes, font properties, and spacing.
Do well to also make it responsive as it is currently not responsive. ensure the popup on the mobile screen matches the given design😉.